macos/Classes/BiometricStorageImpl.swift

```diff
@@ -289,7 +281,8 @@ class BiometricStorageFile {
       kSecValueData as String: content.data(using: String.Encoding.utf8) as Any,
     ]) { (_, new) in new }
     var status = SecItemAdd(query as CFDictionary, nil)
-    if (status == errSecDuplicateItem) {
+    
+    if (status == errSecDuplicateItem || initOptions.authenticationDevicePinFallback) {
```

Comment:
these 2 seem unrelated. Did you test the case where we don't have a value and we use pin fallback?
Wouldn't `query.removeValue` crash?

Reply:
This change is a cosmetic one. Apple don't show the biometric challenge on SecItemAdd. QA dept however needs to have it in the recovery flow, that's why I added the update where it is shown. 
When we move 100% to keyAuth this will be removed.